pom.xml

```diff
@@ -249,13 +249,8 @@ under the License.
                       ! https://issues.apache.org/jira/browse/MINVOKER-174
                       -->
                     <setupIncludes>
-                      <setupInclude>setup_jar_module/pom.xml</setupInclude>
-                      <setupInclude>setup_jar_automodule/pom.xml</setupInclude>
-                      <setupInclude>setup_x/pom.xml</setupInclude>
+                      <setupInclude>setup*/pom.xml</setupInclude>
```

Comment:
This makes sure that any new `setup` projects following the naming convention will be automatically picked up. If there is any potential concern with this approach, I can revert to the explicit list.